From af762e43424382d895d3a10aa6632adc601fdbad Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Thu, 5 Mar 2015 10:25:46 +0000
Subject: [PATCH] arm: Use -M virt on 32 bit ARM, don't pass -dtb parameter.

Note this means all the dtb code in libguestfs & supermin is
obsolete/unused, and may be deleted at some point in future.
---
 src/guestfs-internal.h | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/src/guestfs-internal.h b/src/guestfs-internal.h
index 64bad70..43cdd4d 100644
--- a/src/guestfs-internal.h
+++ b/src/guestfs-internal.h
@@ -151,12 +151,7 @@
 
 /* Machine types.  XXX Make these configurable. */
 #ifdef __arm__
-/* In future, we can use mach-virt and drop the device tree completely
- * (because the plan is for qemu to entirely generate the device tree
- * internally when using mach-virt).
- */
-#define MACHINE_TYPE "vexpress-a15"
-#define DTB_WILDCARD "vexpress*a15-tc1.dtb"
+#define MACHINE_TYPE "virt"
 #endif
 #ifdef __aarch64__
 #define MACHINE_TYPE "virt"
-- 
2.3.1

